docs/dev/linters.md

```diff
@@ -45,7 +45,7 @@ A: This could have several causes:
     - `go version` should output the same as [the repository .go-version](https://github.com/DataDog/datadog-agent/blob/dffd3262934a5540b9bf8e4bd3a743732637ef37/.go-version).
     - `golangci-lint --version` should output the same as [the repository internal/tools/go.mod](https://github.com/DataDog/datadog-agent/blob/dffd3262934a5540b9bf8e4bd3a743732637ef37/internal/tools/go.mod/#L8).
 - You're testing OS specific code; running locally, the linters only get the results for your local OS.
-- You didn't run `inv tidy-all` in the repository, making some dependencies in the remote env outdated compared to your local env.
+- You didn't run `deva tidy` in the repository, making some dependencies in the remote env outdated compared to your local env.
```

Comment:
Just a double check, you removed the `-all` part, if it was intentional it's ok

Reply:
It is I deprecated it in favour of `tidy` 🙂